### PR TITLE
webdriver: add preliminary tests for Get Current URL command

### PR DESCRIPTION
**Pull request by andreastt**

### CODE REVIEW

**Comment by andreastt** [comment]
@lukeis Do you want to have a look at this too?

File: webdriver/navigation.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+def test_get_current_url_file_protocol(session):
+    # tests that the browsing context remains the same
+    # when navigated privileged documents
+    session.url = "file:///"
+    assert session.url == "file:///"
+
+
+def test_get_current_url_malformed_url(session):
+    session.url = "foo"
+    assert session.url
```
Comment (lukeis) [inline webdriver/navigation.py]: shouldn't you be asserting an expected value here? maybe first doing session.url = "about:blank" and verifying that it remains?

File: webdriver/navigation.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    assert isinstance(body["value"], types.StringTypes)
+
+
+def test_get_current_url_special_pages(session):
+    session.url = "about:blank"
+    assert session.url == "about:blank"
+
+
+# TODO(ato): This test requires modification to pass on Windows
+def test_get_current_url_file_protocol(session):
```
Comment (lukeis) [inline webdriver/navigation.py]: do you intend to add other protocols? like ftp? and http vs. https?

Comment (andreastt) [inline webdriver/navigation.py]: We could do, actually. I’ll add a TODO for that.

The primary reason for visiting file:// is that it’s considered a “protected” non-scriptable document in most browsers, and we want to make sure that drivers can access them.

File: webdriver/navigation.py (hunk 1 above)
Comment (andreastt) [inline webdriver/navigation.py]: I think what browsers do when on `window.location = "foo"` is rather unspecified, e.g. some may redirect you to a search engine and others might show you an error page. But we know they _shouldn’t_ fail. So whatever the URL we end up on is, we just ensure we can still use the session.

File: webdriver/navigation.py (hunk 2 above)
Comment (andreastt) [inline webdriver/navigation.py]: Addressed in 418fcc9

**Comment by lukeis** [comment]
👍

**Comment by andreastt** [comment]
Thanks!

**Pull request merged**